content/en/docs/tasks/extend-kubectl/kubectl-plugins.md

```diff
@@ -10,131 +11,266 @@ content_template: templates/task
 
 {{< feature-state state="alpha" >}}
 
-This guide shows you how to install and write extensions for [kubectl](/docs/user-guide/kubectl/). Usually called *plugins* or *binary extensions*, this feature allows you to extend the default set of commands available in `kubectl` by adding new subcommands to perform new tasks and extend the set of features available in the main distribution of `kubectl`.
+This guide shows you how to install and write extensions for [kubectl](/docs/reference/kubectl/kubectl/). By thinking of core `kubectl` commands as essential building blocks for interacting with a Kubernetes cluster, we can think
+of plugins as a means of utilizing these building blocks to create more complex behavior. Plugins extend `kubectl` with new sub-commands, allowing for new and custom features not included in the main distribution of `kubectl`.
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
 You need to have a working `kubectl` binary installed. 
 {{< note >}}
-**Note:** Plugins were officially introduced as an alpha feature in the v1.8.0 release. So, while some parts of the plugins feature were already available in previous versions, a `kubectl` version of 1.8.0 or later is recommended.
+**Note:** Plugins were officially introduced as an alpha feature in the v1.8.0 release. They have been re-worked in the v1.12.0 release to support a wider range of use-cases. So, while some parts of the plugins feature were already available in previous versions, a `kubectl` version of 1.12.0 or later is recommended if you are following these docs.
```

Comment:
nit: Possibly remove `So`, capitalize while.
Is it, While some parts of the plugins features were already available, or While some parts of the plugins feature was already available in previous versions...?